audit/src/main/scala/tech/beshu/ror/audit/EnvironmentAwareAuditLogSerializer.scala

```diff
@@ -17,8 +17,16 @@
 package tech.beshu.ror.audit
 
 import org.json.JSONObject
+import tech.beshu.ror.audit.AuditSerializationHelper.AuditFieldName
 
 trait EnvironmentAwareAuditLogSerializer {
```

Comment:
I found no implementation of this serializer. 
Now, I'm not sure if we still support the changes described in RORDEV-1414.